frontend/src/index.css

```diff
@@ -139,3 +139,19 @@ https://create-react-app.dev/docs/adding-images-fonts-and-files/
 .vega-border-image {
   border-image: url('./images/vega-bg.png') 15%;
 }
+
+::-webkit-scrollbar {
+  width: 10px;
+}
+
+::-webkit-scrollbar-track {
+  background: var(--light-gray5);
+}
+
+::-webkit-scrollbar-thumb {
+  background: var(--gray1);
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background: var(--dark-gray5);
+}
```

Comment:
I think this is fine, but there's a little we could do for firefox here. It supports a couple of properties ('scrollbar-width' and 'scrollbar-color'). IIRC to even see those on Mac you need to enable scrollbar visibility so it's not gonna help a great number of folks, but it's something!

Reply:
As this is a desktop app, this only runs in webview and doesn't run in FF so no point as nothing else supports these, but good eye!

Reply:
This doesn't run in firefox. It runs on webkit and MS webview.